sorting/test_sort.py

```diff
@@ -2,114 +2,126 @@
+    assert s == []
 
 
 def test_list_one_element_insert_sort():
     """Test insert sort on one element."""
-    from sortings import Sortings
-    s = Sortings([1])
-    s.insert_sort()
-    assert s.sort_list == [1]
+    from insert_sort import insert_sort
+    s = insert_sort([1])
+    assert s == [1]
 
 
 def test_simple_list_insert_sort():
     """Test insert sort on basic list."""
-    from sortings import Sortings
-    s = Sortings([5, 4, 3, 2, 1])
-    s.insert_sort()
-    assert s.sort_list == [1, 2, 3, 4, 5]
+    from insert_sort import insert_sort
+    s = insert_sort([5, 4, 3, 2, 1])
+    assert s == [1, 2, 3, 4, 5]
 
 
 def test_insert_sort_random_list():
-    """Test random list of integers."""
-    from sortings import Sortings
-    x = [randint(0,9) for p in range(0, 9)]
-    s = Sortings(x)
+    """Test insert sort on random list of integers."""
+    from insert_sort import insert_sort
+    x = [randint(0, 9) for p in range(0, 9)]
+    s = insert_sort(x)
     x.sort()
-    assert s.sort_list == x
+    assert s == x
 
 
-def test_quick_sort_empty_list():
-    """Quick sort on empty list."""
-    from sortings import Sortings
-    s = Sortings()
-    x = s.quick_sort([])
-    assert x == []
+def test_merge_sort_empty_list():
+    """Test merge sort on empty list."""
+    from merge_sort import merge_sort
+    s = merge_sort([])
+    assert s == []
 
 
-def test_quick_sort_list_len_one():
-    """Quick sort on one element list."""
-    from sortings import Sortings
-    s = Sortings()
-    x = s.quick_sort([1])
-    assert len(x) == 1
+def test_list_one_element_merge_sort():
+    """Test merge sort on one element."""
+    from merge_sort import merge_sort
+    s = merge_sort([1])
+    assert s == [1]
 
 
-def test_quick_sort_simple_list():
-    """Quick sort on basic list."""
-    from sortings import Sortings
-    s = Sortings()
+def test_merge_sort_simple_list():
+    """Test Merge sort on basic list."""
+    from merge_sort import merge_sort
     x = [10, 5, 7, 2, 12]
-    k = s.quick_sort(x)
+    s = merge_sort(x)
+    x.sort()
+    assert s == x
+
+
+def test_merge_sort_rand_list():
+    """Test merge sort on random list."""
+    from merge_sort import merge_sort
+    import random
+    x = random.sample(range(100), 10)
+    k = merge_sort(x)
     x.sort()
     assert k == x
 
 
+def test_quick_sort_empty_list():
+    """Test Quick sort on empty list."""
+    from quicksort import quick_sort
+    x = quick_sort([])
+    assert x == []
+
+
+def test_quick_sort_list_len_one():
+    """Test Quick sort on one element list."""
+    from quicksort import quick_sort
+    x = quick_sort([1])
+    assert len(x) == 1
+
+
 def test_quick_sort_rand_list():
     """test quick sort on random list."""
-    from sortings import Sortings
+    from quicksort import quick_sort
     import random
-    s = Sortings()
     x = random.sample(range(100), 10)
-    k = s.quick_sort(x)
+    k = quick_sort(x)
     x.sort()
     assert k == x
```

Comment:
This is only sorting one actual list of numbers. You can write more tests or use parametrization to show that you can sort a variety of lists of numbers.